en/TOC.md

```diff
@@ -29,7 +29,13 @@
   - [Destroy a TiDB cluster](destroy-a-tidb-cluster.md)
   - [Restart a TiDB Cluster](restart-a-tidb-cluster.md)
   - [Maintain a Hosting Kubernetes Node](maintain-a-kubernetes-node.md)
-  - [Backup and Restore](backup-and-restore.md)
+  + Backup and Restore
+    - [Helm Charts-Based Backup and Restoration](backup-and-restore-using-helm-charts.md)
+    + CRD-Based Backup and Restoration
+      - [Back up Data to GCS](backup-to-gcs.md)
+      - [Restore Data From GCS](restore-from-gcs.md)
+      - [Back up Data to S3-Compatible Storage](backup-to-s3.md)
+      - [Restore Data From S3-Compatible Storage](restore-from-s3.md)
```

Comment:
is there any reason why "to" and "up" are in lower-case but "From" is capitalized?

Reply:
Thanks for pointing it out. According to the [APA style guide](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case) on title case capitalization, words of four letters or more should be capitalized.

Reply:
Okay 🙂

Reply:
Sorry, I've confirmed with my team that our documentation adopts the Chicago Style, which means that `from` should not be capitalized.